pkg/project/project.go

```diff
@@ -55,6 +56,11 @@ func Create(client *occlient.Client, projectName string) error {
 
 // Delete deletes the project with name projectName and sets the project with name currentProject as current project and returns errors if any
 func Delete(client *occlient.Client, projectName string) error {
+
+	// Loading spinner
+	s := util.LogStatusf("Deleting project %s", projectName)
+	defer s.End(false)
```

Comment:
hmmm So now each log statement involves 2 lines

Reply:
Only for status / spinner since the spinner is in it's own Go routine